In [1]:
import numpy as np

In [2]:
from statsmodels.stats.weightstats import ztest

In [8]:


# Dados da amostra
altura_turma = np.array([165, 157, 163, 170, 167, 161, 160, 170, 157, 160, 157, 168, 162, 170, 167, 165, 166, 158, 174, 169, 163, 174, 162, 170, 155, 154, 158, 160, 166]) #29 ALUNAS 
media_nacional = 161
alfa = 0.05
# Teste z para uma amostra
z_statistic, p_value = ztest(altura_turma, value=media_nacional)
print("Teste z para uma amostra:")
print(f"z-statistic: {z_statistic:.4f}")
print(f"p-value: {p_value:.4f}")

Teste z para uma amostra:
z-statistic: 2.6201
p-value: 0.0088


In [9]:
if p_value < alfa:
    print("Rejeitamos a hipótese nula. Há evidências estatisticamente significativas para afirmar que a altura média da turma nao é igual à altura média nacional da população feminina brasileira.")
else:
    print("Não rejeitamos a hipótese nula. Não há evidências estatisticamente significativas para afirmar que a altura média da turma nao é igual à altura média nacional da população feminina brasileira.")

Rejeitamos a hipótese nula. Há evidências estatisticamente significativas para afirmar que a altura média da turma nao é igual à altura média nacional da população feminina brasileira.


**Queremos avaliar se a altura média da turma on35 é significativamente diferente da altura média nacional da população feminina brasileira, que é de 160 cm.**

**Hipótese Nula (H0):**
H0​: A altura média da turma on 35 é igual à altura média nacional.
H0:μ=160

**Hipótese Alternativa (H1):**
H1​: A altura média dos alunas da turma é diferente da altura média nacional.
H1:μ≠160



**A altura média da turma é significativamente diferente de 160 cm?** SIM


**Conclusão:**

O valor p (0.0088) é menor que 0.05. Portanto, rejeitamos a hipótese nula.

**Interpretação Final:**

A altura média da turma on35 nao é igual à altura média nacional da população feminina brasileira, que é de 160 cm.
 Em outras palavras, há evidências estatisticamente significativas para afirmar que a altura média da turma nao é igual à altura média nacional da população feminina brasileira.